### PR TITLE
[TASK] Env Checker

### PR DESCRIPTION
**Pull request by SlawomirBoczek**
Magerun pull-request check-list:

- [x] Pull request against develop branch (if not, just close and create a new one against it)
- [x] README.md reflects changes (if any)

Subject: Env Checker

Fixes: None.

Changes proposed in this pull request:
So this is the pull request about Env Checker that verifies the env.php file and checks if all required keys exist, and if those have correct values (this just checks some of them). I am open to any suggestions, as the official documentation lacks a bit of detailed description of those values, you might have some better ideas to this.

### CODE REVIEW

**Comment by cmuench** [comment]
@SlawomirBoczek  Thanks for contributing to n98-magerun2. I will check your code after my holidays.

**Comment by cmuench** [comment]
@SlawomirBoczek Thanks for Contribution of the new checks.

File: config.yaml [hunk 1]
```diff
@@ -454,3 +458,36 @@ commands:
...
+        - cache_types.db_ddl
+        - cache_types.eav
+        - cache_types.customer_notification
+        - cache_types.config_integration
+        - cache_types.config_integration_api
+        - cache_types.full_page
+        - cache_types.translate
+        - cache_types.config_webservice
+        - cache_types.compiled_config
+        - install.date
```
Comment (ktomk) [inline config.yaml:493]: I've seen you've been taking care of white-space. Please note as well the missing Newline at the end of file here.

Comment (cmuench) [reply config.yaml:493]: @ktomk Thanks for the hint. Will check that before we release the next version.

Comment (cmuench) [reply config.yaml:493]: @ktomk Checked that. Seems that for YAML files it's unusal to add a newline at the end. But I did not find a hint in the official specification.

Comment (ktomk) [reply config.yaml:493]: @cmuench YAML has no such requirement, it's just POSIX or UNIX text files which aligns very well with YAML. Also git normally highlights those missing, nowadays Github, too.

**Pull request merged**